Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic.yml

```diff
@@ -1,6 +1,8 @@
-id: detonate_url_-_generic
+id: Detonate URL - Generic_patched
```

Comment:
We cannot change IDs of playbooks, that would break BC. Usually the ID would be the same as the Name, but we do have legacy content where that is not the case, and we can never change it.

Also a best practice is to use `_dev` or `_copy` suffixes for playbooks, because Demisto SDK knows how to format and validate those.